cosmos/config.py

```diff
@@ -0,0 +1,122 @@
+"""Module that contains all Cosmos config classes."""
+
+from __future__ import annotations
+
+import shutil
+from dataclasses import dataclass, field
+from pathlib import Path
+from logging import getLogger
+
+from cosmos.constants import TestBehavior, ExecutionMode, LoadMode
+from cosmos.exceptions import CosmosValueError
+
+logger = getLogger(__name__)
+
+
+@dataclass
+class RenderConfig:
+    """
+    Class for setting general Cosmos config.
+
+    :param emit_datasets: If enabled test nodes emit Airflow Datasets for downstream cross-DAG
+    dependencies. Defaults to True
+    :param test_behavior: The behavior for running tests. Defaults to after each
```

Comment:
```suggestion
    :param test_behavior: The behavior for running tests. Defaults to after each (model)
```

Reply:
We can log a ticket and assign it to a future milestone to change where the datasets are emitted. I do think we need to review how we're supporting tests overall, including running a test per node, if the user wants to.